### DISCUSSION TITLE
GroupedVirtuoso sticky 'virtuoso-top-item-list' is last in order within 'virtuoso-scroller'. Possible to change somehow?

### DISCUSSION THREAD

**Discussion by lifter035** [petyosi/react-virtuoso#996] [Q&A] [answered]
Hi,

I am working on accessibility and GroupedVirtuoso headers also have selection checkboxes to select/deselect the whole group.
Unfortunately top group header which is housed in 'virtuoso-top-item-list' is last in dom so while tabbing through interactive elements that sticky header is actually last in the tab order.

Is there a way to configure this so that 'virtuoso-top-item-list' is first in dom within 'virtuoso-scroller'?

Yes I know I can add tabIndex values to all headers and items within groups (they also have selection checkboxes) but that's not really advisable when doing accessibility.

Thank you in advance,

Eddie
<img width="834" alt="image" src="https://github.com/petyosi/react-virtuoso/assets/9085424/50e398fc-03a6-469c-90a0-65390e04039f">

**Comment by petyosi** [comment, marked as answer]
Unfortunately, no. Not without reworking the measurements logic of the component. It's a mistake I've made when doing the HTML structure, but it's too costly to fix.

**Reply by lifter035** [reply]
Thanks for the prompt reply Petyo. Will try and work around it

**Reply by lifter035** [reply]
I actually managed to swap their order using scrollerRef and waiting for both sticky and viewport elements to get rendered.
Everything works as expected with multipage dynamic data and infinite scroll
